cluster/addons/node-problem-detector/npd.yaml

```diff
@@ -26,28 +26,28 @@ subjects:
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
-  name: npd-v0.8.0
+  name: npd-v0.8.5
   namespace: kube-system
   labels:
     k8s-app: node-problem-detector
-    version: v0.8.0
+    version: v0.8.5
     kubernetes.io/cluster-service: "true"
     addonmanager.kubernetes.io/mode: Reconcile
 spec:
   selector:
     matchLabels:
       k8s-app: node-problem-detector
-      version: v0.8.0
+      version: v0.8.5
   template:
     metadata:
       labels:
         k8s-app: node-problem-detector
-        version: v0.8.0
+        version: v0.8.5
         kubernetes.io/cluster-service: "true"
     spec:
       containers:
       - name: node-problem-detector
-        image: k8s.gcr.io/node-problem-detector:v0.8.0
+        image: k8s.gcr.io/node-problem-detector:v0.8.5
```

Comment:
It seems this image (or `k8s.gcr.io/node-problem-detector/node-problem-detector:v0.8.5`) is not yet available.

Reply:
Yup, that's why I put this PR on hold in the first place :).